Lazydocs: API Reference doc refresh (#1424)
wandb/docs: content/en/guides/models/app/features/panels/run-comparer.md

- To copy any configuration or metric's value, hover your mouse over the value, then click the copy button. The entire value is copied, even if it is too long to display on the screen.

{{% alert %}}

> By default, Run Comparer does not differentiate runs with different values for [`job_type`]({{< relref "/ref/python/sdk/functions/init" >}}). This means that it is possible to compare runs that are not comparable within a project. For example, you could compare a training run to a model evaluation run. A training run could contain run logs, hyperparameters, training loss metrics, and the model itself. An evaluation run could use the model to check the model's performance on new training data.

Reviewer: ```suggestion
By default, Run Comparer does not differentiate runs with different values for [`job_type`]({{< relref "/ref/python/sdk/functions/init" >}}). This makes it possible to compare runs that are not directly comparable. For example, you could compare a training run to a model evaluation run. A training run could contain run logs, hyperparameters, training loss metrics, and the model itself. An evaluation run could use the model to check the model's performance on new training data.
```
